static/src/stylesheets/_vars.scss

```diff
@@ -136,6 +136,18 @@ $sport-away:             #003f64;
 
 // Stati
```

Comment:
This might be my favourite pull request comment ever

![image](https://cloud.githubusercontent.com/assets/11922253/17368021/4dcb4d1e-598a-11e6-9e98-feced12c82e3.png)